UI/Web/src/app/book-reader/_components/reader-settings/reader-settings.component.html

```diff
@@ -70,6 +70,15 @@ <h2 class="accordion-header">
                         <span class="phone-hidden">&nbsp;{{readingDirectionModel === ReadingDirection.LeftToRight ? 'Left to Right' : 'Right to Left'}}</span>
                     </button>
                 </div>
+                <div class="controls" style="display: flex; justify-content: space-between; align-items: center; ">
+                    <label id="writingstyle" class="form-label">Writing Style <i class="fa fa-info-circle" aria-hidden="true" placement="top" [ngbTooltip]="writingStyleTooltip" role="button" tabindex="0" aria-describedby="writingStyle-help"></i></label>
```

Comment:
this should be writing-style. For any ids in html, it should be kebab case.

Reply:
You also need a span with writingStyle-help as an id so the screenreader announces what's in the tooltip. Look around and you should find an example